src/cmd/linuxkit/gcp.go

```diff
@@ -227,7 +233,14 @@ func (g GCPClient) CreateInstance(name, image, zone, machineType string, disks D
 		} else {
 			diskSizeGb = int64(convertMBtoGB(disk.Size))
 		}
-		diskOp, err := g.compute.Disks.Insert(g.projectName, zone, &compute.Disk{Name: diskName, SizeGb: diskSizeGb}).Do()
+		diskObj := &compute.Disk{Name: diskName, SizeGb: diskSizeGb}
+		if vtpm {
+			// TODO: We could/should check here if the image has UEFI_COMPATIBLE feature
```

Comment:
I was going to write that, and then saw your comment. How hard is it to check? Conversely, if it is _not_ compatible, will it fail a compatibility check immediately when you call `g.compute.Instances.Insert()` [here](https://github.com/linuxkit/linuxkit/pull/3790/files#diff-2ea55b1d08ccdd78b92f240afc0629a11c3230ee7694059b001a1dfacca09c4cR300)? Or is that one of the things that only fails asynchronously? Or does it not fail at all?

If the call below to `Insert()` will fail right away, then no real need to check it here; that is good enough.